inst/qml/msaType1Gauge.qml

```diff
@@ -113,48 +114,48 @@ Form
 
 		CheckBox
 		{
-			name: 								"biasRun"
+			name: 								"runChart"
 			label: 								qsTr("Run chart")
 			checked: 							true
 
 			CheckBox
 			{
-				name: 							"biasRunDots"
+				name: 							"runChartIndividualMeasurementDots"
 				label: 							qsTr("Display individual measurements")
 				checked: 						true
 			}
 
 			CheckBox
 			{
-				name: 							"biasRunTolLims"
+				name: 							"runChartToleranceLimitLines"
 				label: 							qsTr("Display tolerance limits")
 				checked: 						true
 			}
 		}
 
 		CheckBox
 		{
-			name: 								"biasHistogram"
+			name: 								"histogram"
 			label: 								qsTr("Histogram")
 
 			DropDown
 			{
-				id: 							binWidthType
-				name:							"biasBinWidthType"
+				name:							"histogramBinWidthType"
 				label:							qsTr("Bin width type")
+				id: 							binWidthType
 				indexDefaultValue:				0
 				values: [
 					{ label: qsTr("Sturges"), value: "sturges"},
 					{ label: qsTr("Scott"), value: "scott"},
 					{ label: qsTr("Doane"), value: "doane"},
-					{ label: qsTr("Freedman-Diaconis"), value: "fd"},
+					{ label: qsTr("Freedman-Diaconis"), value: "freedman-diaconis"},
```

Comment:
```suggestion
					{ label: qsTr("Freedman-Diaconis"), value: "freedmanDiaconis"},
```

Reply:
This is directly forwarded to `hist(breaks = ...)`, which takes "fd" and "freedman-diaconis". If we really want "freedmanDiaconis", I'd have to write some code in the function that changes it back to "fd" or "freedman-diaconis". Should we do this?

Reply:
yes unfortunately we will need to do this. The reasoning is that consistency of the syntax within JASP has a priority over consistency of the JASP module with the underlying imlementation of the analysis in R

Reply:
Okay, no problem!